.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.4'
```

Comment:
Just to understand this correctly - it's in `.devcontainer` ... and it's intended to be used ONLY by vscode when running the remote container configuration?

so when i want to run a docker-container for something else, i'd use a different compose file (at the project root) ... ?

Reply:
> Just to understand this correctly - it's in .devcontainer ... and it's intended to be used ONLY by vscode when running the remote container configuration?

So .devcontainer is a convention that come from vscode i think. I have now slit in built 11b7521 that users can built dev docker image and use with or without vscode (with multistage built).

> so when i want to run a docker-container for something else, i'd use a different compose file (at the project root) ... ?

Here .devcontainer folder contain only what is needed for dev. But at the root you can create a dockerfile and docker-compose if needed for production environment.

> while i do use vscode (almost exclusively, apart from VIM) - i never used remote-containers before.

For me it was a life changer :-) I only develop this way now.